microsetta_private_api/api/microsetta_private_api.yaml

```
@@ -862,6 +862,214 @@ paths:
           # NB: This error would cover the case where you can't dissociate it bc it was already processed
           $ref: '#/components/responses/422UnprocessableEntity'
 
+  '/accounts/{account_id}/sources/{source_id}/samples/{sample_id}/vioscreen/session':
+    get:
+      operationId: microsetta_private_api.api.read_sample_vioscreen_session
+      tags:
+        - Sample
```

Let's place under a new tag as there is a logical grouping with vioscreen
```suggestion
        - Vioscreen
```